app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -45,13 +51,16 @@ class HomeViewModel(
 ) : ObservableViewModel() {
 
   private val profileId: ProfileId = ProfileId.newBuilder().setInternalId(internalProfileId).build()
+  private val promotedStoryListLimit = activity.resources.getInteger(
+    R.integer.promoted_story_list_limit
+  )
 
   private val profileDataProvider: DataProvider<Profile> by lazy {
     profileManagementController.getProfile(profileId)
   }
 
-  private val ongoingStoryListSummaryDataProvider: DataProvider<OngoingStoryList> by lazy {
-    topicListController.getOngoingStoryList(profileId)
+  private val ongoingStoryListSummaryDataProvider: DataProvider<RecommendedActivityList> by lazy {
```

Comment:
Similar to above, maybe use "Promoted Activity" instead for clarity of what this is meant to represent, since we still us "Promoted List" to describe all these cases (also the "ongoingStoryListSummaryDataProvider" might be changed to "promotedStoryListSummaryDataProvider as well?)

Also elsewhere "ongoing list" or "recommended activity" is used. Since "promoted stories list" is used to describe multiple cases, I think we can keep this description consistent to clarify that these are all related and to reduce confusion with the recommended stories case.

Reply:
Done.